### PR TITLE
Add project wide prettier config

### PR DESCRIPTION
**Pull request by sir-marc**

### CODE REVIEW

**Comment by StuckiSimon** [comment]
Could you rebase the branch?

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "concurrently": "^5.1.0",
     "cross-env": "^7.0.0",
     "cypress": "^4.0.2",
+    "prettier": "^1.19.1",
```
Comment (StuckiSimon) [inline package.json]: "We recommend pinning an exact version of prettier in your package.json as we introduce stylistic changes in patch releases."

**Review by StuckiSimon — CHANGES_REQUESTED** [review]
We must also make sure to allow eslint and prettier to co-exist.

File: package.json (hunk 1 above)
Comment (sir-marc) [reply package.json]: #classic

**Review by StuckiSimon — CHANGES_REQUESTED** [review]
Don't forget to move the dependencies & scripts in package.json as well.

**Pull request closed without merging**